packages/core/types/src/index.js

```diff
@@ -0,0 +1,8 @@
+// @flow strict-local
+
+import type WorkerFarm from '@parcel/workers';
```

Comment:
Does this mean `@parcel/types` still has a dependency on `@parcel/workers`? Does that bring in any other heavy deps?

Reply:
It does not bring heavy dependencies (that should be fine) but yes